README.md

```diff
@@ -1,16 +1,18 @@
 # Py-Elotl
 
-Python package for Natural Language Processing (NLP), focused on low-resource languages spoken in Mexico.
+Python package for Natural Language Processing (NLP), focused on low-resource
+languages spoken in Mexico.
 
 This is a project of [Comunidad Elotl](https://elotl.mx/).
 
 Developed by:
 - Paul Aguilar [@penserbjorne](https://github.com/penserbjorne), [paul.aguilar.enriquez@hotmail.com](mailto:paul.aguilar.enriquez@hotmail.com)
 - Robert Pugh [@Lguyogiro](https://github.com/Lguyogiro), [robertpugh408@gmail.com](mailto:robertpugh408@gmail.com)
+- Diego Barriga [@umoqnier](https://github.com/umoqnier/), [dbarriga@ciencias.unam.mx](mailto:dbarriga@ciencias.unam.mx)
 
-Requiere python>=3.X
+Requiere python>=3.6
 
-- Development Status `Pre-Alpha`. Read [Classifiers](https://pypi.org/classifiers/)
+- Development Status `Alpha`. Read [Classifiers](https://pypi.org/classifiers/)
```

Comment:
I think that the project is mature enough to change the status to `Alpha`. What do you think?

Reply:
yes, this sounds good to me.